dist/index.js

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env node
```

Comment:
This file shouldn't be committed - let's use the npm `prepare` lifecycle to generate it on install.